build2cmake/src/cmake/utils.cmake

```
@@ -503,5 +503,22 @@ function (define_gpu_extension_target GPU_MOD_NAME)
     target_link_libraries(${GPU_MOD_NAME} PRIVATE ${TORCH_LIBRARIES})
   endif()
 
+  # if `LOW-MEM` keep every big compile/link in a 1-slot pool
+
+  # even with -j1, the generator might start a second huge link
+  # while the first is still in memory. Assigning every GPU module to a 
+  # dedicated 1-slot pool prevents that.
+  if(LOW_MEM)
```

Configurable bools usually use `ENABLE_<name>`.

```suggestion
  if(ENABLE_LOW_MEM)
```

Should probably also go in a different file, we should keep `utils.cmake` as close to vLLM as possible, so that we can periodically re-vendor. Maybe it could be a separate function that is also applied to the target?

---

removed all low memory flags from this PR as it should probably be its own PR if needed